### ISSUE TITLE
[ACCESSIBILITY] Missing Alt Text on Athletics Upcoming Events 

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#291]
Describe the accessibility issue
As a screen reader user, I should hear a description of what an image is when I am able to click on it.  For Athletics, Upcoming Events, when I reach this screen I hear "Unlabeled image"

To Reproduce
1. Enable screen reading accessibility feature on your given device
2. Navigate to IL app
3. Click on Athletics
4. tap on any event, example "Nebraska at Illinois" 

Expected behavior
As a screen reader user, I should hear a description of what an image is when I am able to click on it.  For Athletics, Upcoming Events, when I reach this screen, I should hear something descriptive about image, i.e. "Image of three men celebrating after scoring a point"

Screenshots
![Image needs alt text](https://user-images.githubusercontent.com/82892873/121958086-e045f880-cd28-11eb-875e-aaa57988dc4b.PNG)

Smartphone (please complete the following information):

Samsung Galaxy S9 Android 10
Additional context

**Comment by acovher2** [comment]
![Screenshot_20210614-152206_Illinois](https://user-images.githubusercontent.com/82892873/121959212-19cb3380-cd2a-11eb-808e-57aaced18aba.jpg)
This image is also missing alt text.

**Comment by TodorBachvarov** [comment]
We are not able to add custom description for each image directly from the client, as these images are comming from the backend and we do not know what will be displayed in this image. If we want to do this we will have to provide this description from the backend as a property of the Event model. So for now the only thing that we can do is to remove the "Unlabeled image" pronouncement.

**Comment by acovher2** [comment]
If the client is uploading these images on the backend with other content entry then they need a required field to provide their own custom alt text for images for them to be accessible.  Removing "unlabeled image" from being pronounced does **not** resolve the issue, please do not do that. 
Thank you

**Comment by TodorBachvarov** [comment]
Updated to pronounce "Sports Image"

**Comment by acovher2** [comment]
This works for this image only. Should I submit future request for client to get backend Alt text entry field which is required for any photo upload? This is really what needs done if client can upload custom images in their backend. Please let me know. Thank you.

**Comment by acovher2** [comment]
Closing.
